app/Http/Controllers/QuestionAPIController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Question;
+use App\Models\Quiz;
+use Exception;
+use Illuminate\Http\Request;
+
+class QuestionAPIController extends Controller
+{
+    function addQuestion(Request $request)
+    {
+        $request->validate([
+            'question' => 'required|string|max:128',
+            'points' => 'required|integer|max:10',
+            'quiz_id' => 'required|integer|exists:quizzes,id',
+            'hint' => 'string|max:128',
+        ]);
+
+        try {
+            $quiz = Quiz::find($request->quiz_id);
+            $question = new Question();
+            $question->question = $request->question;
+            $question->points = $request->points;
+            $question->quiz_id = $request->quiz_id;
+
+            if ($request->hint) {
+                $question->hint = $request->hint;
+            }
```

Comment:
Did you try it without this guarding `if`? What result did you get? I thought laravel automagically accounted for fields not being present